cloud-service-providers/azure/aks/setup/README.md

```diff
@@ -0,0 +1,48 @@
+# Setup AKS
+
+The keys to create AKS for NIM is to create proper GPU nodepool.  Details are in the following table. 
```

Comment:
Can we re-word this to make it more clear how the user can use the below table when deploying NIM?